### DISCUSSION TITLE
Neue Platine SafeTech+

### DISCUSSION THREAD

**Discussion by smooty1970** [smarthausleben/ioBroker.wamo#34] [General]
Ich habe einen SYR SafeTech+ mit neuer Platine (jetzt auch mit LAN Anschluss) und vom Hersteller die Doku mit den neuen Befehlen erhalten:
[4_TRIO Befehle zu lokalen Steuerung.pdf](https://github.com/smarthausleben/ioBroker.wamo/files/14162440/4_TRIO.Befehle.zu.lokalen.Steuerung.pdf)
[3_TRIO Parameter fuer lokale Abfrage.pdf](https://github.com/smarthausleben/ioBroker.wamo/files/14162441/3_TRIO.Parameter.fuer.lokale.Abfrage.pdf)

**Comment by smarthausleben** [comment]
Hallo smooty1970,

sorry für die sehr späte Antwort, aber ich habe da wohl das Benachrichtigungs-Email von Github übersehen 🤔

Vielen Dank für die Dokumente. So auf den ersten Blick könnte die Implementierung, über eine entsprechende Einstellung in der Adapter-Konfiguration, relativ einfach möglich sein. 

 Testen könnte ich das allerdings nicht wirklich, da ich dieses Gerät ja nicht selbst habe. Auch mein Ansprechpartner bei SYR meldet sich leider nicht mehr auf meine Anfragen. Hast du da einen anderen Ansprechpartner als Herr B*****ans?

**Reply by smooty1970** [reply]
info@syr.de
oder
Danilo.Kruschwitz@syr.de

im MS Loxone funktioniert die Abfrage ohne Probleme:
http://xxx.xxx.xxx.xxx:5333/trio/get/all
"getBAT":\v

**Reply by smarthausleben** [reply]
Herrn Kruschwitz kannte ich noch nicht. Das wäre dann jetzt mein dritter Ansprechpartner bei SYR 😂. Vielen Dank für die Info.

Dann werde ich da gleich mal Kontakt aufnehmen und fragen wie es mit der Firmware rund um die Floor-Sensoren steht. Da ich an dieser Stelle mit dem Vorgänger von Herr Kruschwitz nicht weiter gekommen bin, hatte das Projekt leider an Priorität verloren.

Davon ausgehend, dass die Parameter und Kommandos die selben geblieben sind, unterscheidet sich der Link ja nur dahingehend, dass aus `.../safe-tec/...` bzw. `.../floorsensor/...` nun `.../trio/...` wird.
